src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,178 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final String FIFTEEN_CHARS_LOGIN = "Traineeeeeeeeee";
+    private static final String SIX_CHARS_LOGIN = "Taylor";
+    private static final String SEVENTEEN_CHARS_LOGIN = "GeorgeTheMagician";
+    private static final String SEVEN_CHARS_LOGIN = "abcabcc";
+    private static final String EIGHT_CHARS_PASSWORD = "password";
+    private static final String SIX_CHARS_PASSWORD = "Swifty";
+    private static final String NINE_CHARS_PASSWORD = "333666999";
+    private static final String SYMBOL_SIX_CHARS_PASSWORD = "!@#$%^";
+    private static final int TWENTY_FIVE_AGES = 25;
+    private static final int EIGHTEEN_AGES = 18;
+    private static final int NINETEEN_AGES = 19;
+    private static final int NINETY_AGES = 90;
+    private static final String THREE_CHARS_LOGIN = "Max";
+    private static final String FIVE_CHARS_EDGE_LOGIN = "marty";
+    private static final String THREE_CHARS_PASSWORD = "dnd";
+    private static final String FIVE_CHARS_EDGE_PASSWORD = "rumba";
+    private static final int NEGATIVE_AGE = -15;
+    private static final int NON_VALID_EDGE_AGE = 17;
+    private static final String EMPTY_FIELD = "";
+    private static final int ZERO_AGE = 0;
+    private static final User FIRST_VALID_USER =
```

Comment:
You don`t need to make users constants, just create method which creates user and pass to it parameters like: createUser(INVALID_LOGN, VALID_NAME....) from signature it will be creat what user will be returned and for which purposes